R/deltaccd_plot.R

```diff
@@ -142,10 +142,12 @@ plotHeatmap = function(geneNames, emat, groupVec = NULL) {
 plotRefHeatmap = function(refCor) {
   if (any(rownames(refCor) != colnames(refCor)) || !isSymmetric(refCor)) {
     stop('refCor must be a correlation matrix, with identical rownames and colnames.')}
+  
+  geneNames = rownames(refCor)
 
   dt = data.table(refCor, gene1 = geneNames)
   dt = data.table::melt(
-    dt, id.vars = 'gene1', measure.vars = 'gene2', value.name = 'rho')
+    dt, id.vars = 'gene1', measure.vars = geneNames, value.name = 'rho')
```

Comment:
This seems like an unrelated change, and I don't see how the new code will work, since line 151 uses a column gene2 that doesn't seem to exist. Have you tested it?

Reply:
I repeat my question.

Reply:
Definitely a typo.

Reply:
That doesn't answer my question, though. Does the function work as is? Have you tested it? Is there a gene2 column?

Reply:
No, it wasn't working. I meant to type `variable.names = 'gene2'`, not `measure.vars`. Also, I needed to change `ematNow` to `refCor`.